### ISSUE TITLE
if target/wasm-bindgen already exists don't attempt to re-add/install it

### ISSUE THREAD

**Issue by ashleygwilliams** [rustwasm/wasm-pack#51]

**Comment by data-pup** [comment]
I would love to help out with this one if it wouldn't be too difficult for a first contribution! Would you have any advice for identifying whether or not target/wasm-bindgen exists already?

I'm guessing that this would mean adding a check to the `cargo_install_wasm_bindgen` function, but I'm not quite sure what the correct method is to identify whether wasm-bindgen has already been added.

**Comment by ashleygwilliams** [comment]
hey @data-pup ! so, i wasn't actually entirely sure- this stack overflow answer seems mostly reasonable: https://stackoverflow.com/questions/35045996/check-if-a-command-is-in-path-executable-as-process. lemme know if you need help digesting it!

**Comment by data-pup** [comment]
Hi! Thank you for the link, that definitely helped. I started working on this today, here's the branch with some first steps. I think I will need to add some extra logic for this to work on Windows, which uses `;` to separate items in $PATH iirc.

Should the progress bar still show up if wasm-bindgen is already installed? Right now it just exits the `cargo_install_wasm_bindgen` function if wasm-bindgen is found in the path.

https://github.com/data-pup/wasm-pack/commit/7b4552081f495e1cb94c3a2ae4febf07fc9a1eb3

**Comment by ashleygwilliams** [comment]
hey @data-pup ! sorry for the delay, i was out sick earlier this week- do you think you could open up a PR? it's much easier to give feedback :D looking forward to reviewing it and thanks so much!

**Comment by data-pup** [comment]
Hi @ashleygwilliams! Sure thing, I can open a PR today. Thanks for helping me out with contributing, I really appreciate it and am looking forward to helping out more!

**Comment by ashleygwilliams** [comment]
@data-pup can you run `rustc --version` and `cargo fmt --version` and let me know what the output is?

**Comment by data-pup** [comment]
@ashleygwilliams Here are the versions. These might be slightly out of date, I can update these and see if that fixes the issue.

```
✨  rustc --version
rustc 1.26.0-nightly (a04b88d19 2018-03-19)

✨  cargo fmt --version
0.3.8-nightly (346238f 2018-02-04)
```

**Comment by fitzgen** [comment]
Fixed in #192
